### ISSUE TITLE
loss be nan after training with several epochs

### ISSUE THREAD

**Issue by JackMing1986** [lucifer443/EfficientDet-Pytorch#5]
bbox_loss is nan, cls_loss is 21.834136962890625
